documentation/src/components/adobe-logo.ts

```diff
@@ -10,7 +10,12 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { LitElement, html, CSSResultArray, property } from 'lit-element';
+import {
+    LitElement,
+    html,
+    CSSResultArray,
+    property,
+} from '@spectrum-web-components/base';
```

Comment:
Originally we moved to this pattern for two reason:
1. Having a central dependency that re-exported the `lit-*` API allowed for us to track the versions thereof in a single place in our repo. Generally, the libraries this track are fairly slow moving, so maybe the benefit here is more ergonomic than practical.
2. Before the `lit@2.0` release, `lit-html`, and the tooling around it, relied heavily on `instanceof` checks that required a shared version across a project. Leveraging a central dependency reduced the changed of package managers accidentally allowing multiple non-matched instances into a project.

I'd say these realities certainly put moving away from this pattern into conversation. I'm not sure that I'd say that reexporting these APIs locks our elements to them in any function manner, however. It would certainly be a breaking change to _stop_ exporting them, but I do see how choosing to basing our elements on another library, as long as the public API to those element remained the same, would be in any way consequential to our consumers. This is one of the reason that, no matter how powerful the pattern may be, we've avoided passing templates into our elements to date. 

Definitely want to discuss this more, so we can get structure a high quality API around this.